src/transformers/models/ernie4_5_vl_moe/video_processing_ernie4_5_vl_moe.py

```diff
@@ -0,0 +1,594 @@
+        # We are downloading both configs because almost all models have a `processor_config.json` but
+        # not all of these are nested. We need to check if it was saved recebtly as nested or if it is legacy style
+        video_processor_dict = None
+        if resolved_processor_file is not None:
+            processor_dict = safe_load_json_file(resolved_processor_file)
+            if "video_processor" in processor_dict:
+                video_processor_dict = processor_dict["video_processor"]
+
+        if resolved_video_processor_file is not None and video_processor_dict is None:
+            video_processor_dict = safe_load_json_file(resolved_video_processor_file)
+
+        if video_processor_dict is None:
+            raise OSError(
+                f"Can't load video processor for '{pretrained_model_name_or_path}'. If you were trying to load"
+                " it from 'https://huggingface.co/models', make sure you don't have a local directory with the"
+                f" same name. Otherwise, make sure '{pretrained_model_name_or_path}' is the correct path to a"
+                f" directory containing a {video_processor_file} file"
+            )
+
+        # Specific to Ernie 4.5 VL Moe, we load the font file along the json (if we draw on frames)
+        draws_on_frames = video_processor_dict.get("draw_on_frames")
+        if (font_name := video_processor_dict.get("font")) is None and draws_on_frames:
+            raise AttributeError(
+                "Expected a `font` to be saved when using `draw_on_frames` in Ernie 4.5 VL Moe; found nothing."
+            )
+        if font_name is not None and draws_on_frames:
+            video_processor_dict["font"] = cached_file(
+                pretrained_model_name_or_path,
+                filename=font_name,
+                cache_dir=cache_dir,
+                force_download=force_download,
+                proxies=proxies,
+                local_files_only=local_files_only,
+                token=token,
+                user_agent=user_agent,
+                revision=revision,
+                subfolder=subfolder,
+                _raise_exceptions_for_missing_entries=False,
+            )
+            try:
+                ImageFont.truetype(video_processor_dict["font"])
+            except (TypeError, OSError):
+                raise OSError(
+                    f"Could not find an associated font file for {video_processor_dict['font']}. "
+                    "Make sure to save a font file along for Ernie 4.5 VL Moe."
+                )
+
+        if is_local:
+            logger.info(f"loading configuration file {resolved_video_processor_file}")
+        else:
+            logger.info(
+                f"loading configuration file {video_processor_file} from cache at {resolved_video_processor_file}"
+            )
+
+        return video_processor_dict, kwargs
```

Comment:
cc @yonigozlan this is something we want to be made easier: processor should define just the file extension / name and from pretrained downloads / finds everything for itt!